pkg/virtctl/vmexport/vmexport.go

```diff
@@ -141,6 +151,8 @@ type VMExportInfo struct {
 	KeepVme        bool
 	IncludeSecret  bool
 	ExportManifest bool
+	PortForward    bool
```

Comment:
Maybe just make this a string and if it is not blank then we know port forward was set.

Reply:
I think your option might be better, I can change it.